### DISCUSSION TITLE
Release build crash on Linux resolved

### DISCUSSION THREAD

**Discussion by kmoks** [SAP-archive/fedem-gui#25] [Announcements]
I found the reason, or at least a workaround, for the immediate crash on startup when build for Release, as commented in #17 and #21. See the top commit [f08346c](https://github.com/kmoks/fedem-gui/commit/f08346c7bdc73aec2e876a423f4d6dad11c0dd5e) on the branch https://github.com/kmoks/fedem-gui/tree/qt6-port. It turns out that Qt6 on Linux wants the argc parameter to be passed as reference (`int& argc`) and not as value (`int argc`).  With this change the crash is gone. On Windows it does not seems to matter though.

@sanguinariojoe it would be great if you could confirm whether this fix also works or not in your build setup.

**Comment by sanguinariojoe** [comment]
It definitely runs...

I still have the same problem though. The SoQt window is rendering nothing...

![image](https://github.com/SAP/fedem-gui/assets/1668392/6f10199d-16b6-4616-8558-2120461ff261)

It is happening even with the simplest SoQt examples. So I will follow anyway my path to get Quarter working.

Thanks!

**Reply by kmoks** [reply]
OK, thanks for the update, and for checking this out. Then your issue is something else then.
